lib/std/crypto/aes/soft.zig

```diff
@@ -435,7 +435,9 @@ const table_decrypt align(64) = generateTable(sbox_decrypt, [_]u8{ 0xb, 0xd, 0x9
 
 // Generate S-box substitution values.
 fn generateSbox(invert: bool) [256]u8 {
-    var array: [256]u8 align(64) = undefined;
+    @setEvalBranchQuota(2000);
```

Comment:
Still not enough :) Try 5000.

Reply:
```suggestion
    @setEvalBranchQuota(5000);
```

It worked on my computer! (TM)

Reply:
CI seems to be passing so far :)